packages/app/src/runner/injectBundle.ts

```diff
@@ -29,26 +24,6 @@ export async function injectBundle () {
     script.onload = () => {
       // just stick config on window until we figure out how we are
```

Comment:
We should move these comments to where this code was moved. This can be simplified to `script.onload = () => resolve()`